language/de/common.php

```diff
@@ -67,6 +67,7 @@
 	'ACCOUNT_DEACTIVATED'			=> 'Dein Benutzerkonto wurde manuell deaktiviert und kann nur durch einen Administrator reaktiviert werden.',
 	'ACP'							=> 'Administrations-Bereich',
 	'ACP_SHORT'						=> 'Adm.',
+	'ACTIVATION_ALREADY_SENT'		=> 'Die Aktivierungs-E-Mail wurde bereits an deine E-Mail-Adresse gesendet. Du kannst sie frühstens erneut nach 24 Stunden anfordern. Solltest du weiterhin Probleme mit der Aktivierung deines Accounts haben, wende dich bitte an die Board-Adminstration.',
```

Comment:
> Du kannst sie frühstens erneut nach 24 Stunden anfordern

Tauschen der Reihenfolge?

> Du kannst sie frühstens nach 24 Stunden erneut anfordern

Reply:
Würde ich auch tauschen

Reply:
Alles klar, mache ich in de_x_sie dann auch so.